psst-gui/src/controller/playback.rs

```diff
@@ -412,11 +412,25 @@ where
                 ctx.set_handled();
             }
             Event::KeyDown(key) if key.code == Code::ArrowRight => {
-                self.next();
+                if key.mods.shift() {
+                    self.next();
+                } else if let Some(now_playing) = &data.playback.now_playing {
+                    let current_position = now_playing.progress;
+                    let max_position = now_playing.item.duration();
+                    let seek_position =
+                        (current_position + Duration::from_secs(10)).min(max_position);
```

Comment:
We should probably make the skip duration configurable - I can see many people wanting that.

Reply:
Good idea!

Reply:
Implemented in the recent commit.